snugget_load.py

```diff
@@ -28,25 +29,57 @@ def main():
     with conn.cursor() as cur:      
       with open(snuggetFile) as csvFile:
         newSnuggets = csv.DictReader(csvFile)
+        rowCount = 1
```

Comment:
Shouldn't this start at 0? You're starting at 1 and then incrementing before you do anything else in the loop.